Assets/MixedRealityToolkit.Examples/Experimental/Dialog/DialogExampleController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Experimental.Dialog;
+
+namespace Microsoft.MixedReality.Examples.Experimental.DialogTest
+{
+    /// <summary>
+    /// This class is used as an example controller to show how to instantiate and launch two different kind of Dialog.
+    /// Each one of the public methods are called by the buttons in the scene at the OnClick event.
+    /// </summary>
+    public class DialogExampleController : MonoBehaviour
+    {
+        [SerializeField]        
+        private GameObject dialogPrefabLarge;
+        public GameObject DialogPrefabLarge
```

Comment:
Please add summary comments describing each property. Also, please add [Tooltip("...")].

This request applies to all serialized fields.